domain/build.gradle.kts

```diff
@@ -35,14 +36,15 @@ android {
 }
 
 dependencies {
-
     implementation(libs.androidx.core.ktx)
     implementation(libs.androidx.appcompat)
     implementation(libs.material)
     testImplementation(libs.junit)
     androidTestImplementation(libs.androidx.junit)
     androidTestImplementation(libs.androidx.espresso.core)
-    implementation("com.google.dagger:hilt-android:2.46") // or the latest version
-    kapt("com.google.dagger:hilt-compiler:2.46")
-    implementation(project(":data"))
+    implementation(libs.hilt.android.v246) // or the latest version
+    kapt(libs.hilt.compiler.v246)
```

Comment:
The comment can be removed and the dependency name mustn't contain its version as well.